released/packages/coq-smlms/coq-smlms.1.0.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+name: "coq-smlms"
+version: "1.0.0"
+maintainer: "Hugo Férée <feree@irif.fr>"
+authors: [
+  "Hugo Férée"
+]
+homepage: "https://gitlab.com/trustworthy-refactoring/coq-smlms"
+dev-repo: "git+https://gitlab.com/trustworthy-refactoring/coq-smlms.git"
+bug-reports: "https://gitlab.com/trustworthy-refactoring/coq-smlms/-/issues"
+license: "MIT"
+build: [
+  [make "-j%{jobs}%"]
+  [make "html"] {with-doc}
+]
+install: [make "install"]
+remove: [
+  [make "uninstall"]
+  ["rmdir" "%{lib}%/coq/user-contrib/AbstractRenaming"]
+]
```

Comment:
Removal is supposed to be handled automatically by opam these days, especially when using coq_makefile.
```suggestion
```